src/components/ContentWidgets/TabView/TabView.css

```diff
@@ -0,0 +1,92 @@
+.tabs-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    list-style-type: none;
+    padding: 1rem;
+    background-color: #420DAB;
+    color: #ffffff;
+}
+
+.tab:hover {
+    color: #DAB420;
+    cursor: pointer;
+}
+
+.current-tab {
+    color: #DAB420;
+    border-bottom: 1px #DAB420 solid;
+}
+
+.current-tab-content, .current-tab-content-editor {
+    width: 100%;
+    border: 1px black solid;
+    margin-bottom: 5px;
+}
+
+.current-tab-content {
+    height: 15rem;
+    overflow: scroll;
+    padding: 3rem;
+    padding-top: 1rem;
+}
+
+.current-tab-content-editor {
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
+}
+
+.current-tab-content-editor > textarea, .current-tab-content > ReactMarkdown {
```

Comment:
huh does this actually work? I would probably use an id here instead or a class `.react-markdown`.